scripts/login_signup.js

```diff
@@ -0,0 +1,134 @@
+import { isValidEmail, isValidPassword } from "./utils/validate.js";
+
+const form = document.forms[0];
+const email = form["email"];
+const nickname = form["nickname"];
+const password = form["password"];
+const passwordConfirm = form["passwordConfirm"];
+const submit = form.lastElementChild;
+
+// 폼 유효성 & 활성화 초기화
+
+let isEmailValid = false;
+let isNicknameValid = false;
+let isPasswordValid = false;
+let isPasswordConfirmValid = false;
+submit.disabled = true;
+
+// 폼 제출 활성화 판단
+
+function updateSubmit() {
```

Comment:
P2:
함수, 변수에 대한 설명 주석은 바로 위에 적어주세요.
```suggestion
// 폼 제출 활성화 판단
function updateSubmit() {
```

Reply:
P2:
button disabled 여부를 js로 컨트롤 하고, html button 요소에 disabled값을 안 주시면 js가 로딩되기 전까지는 버튼이 동작이 가능하니 html button요소에 disbled 속성을 추가해주세요.

Reply:
P2:
해당 파일을 로그인, 회원가입 페이지에서 공용으로 쓰다보니 함수가 복잡해지는 것 같습니다.
login.js, signup.js를 분리하면 해당 함수도 더 단순해질 것 같습니다.